src/plone/testing/publisher.py

```diff
@@ -26,16 +26,16 @@ def setUp(self):
             self.get('configurationContext'))
 
         import zope.security
-        xmlconfig.file('meta.zcml', zope.security, context=context)
+        xmlconfig.file('meta.zcml', zope.security, context=context)  # noqa: D001,E501
```

Comment:
I would suggest avoiding E501 (too long line) be reindenting this:

```
        xmlconfig.file(
            'meta.zcml', zope.security, context=context)  # noqa: D001
```

And why would we ignore D001 (found `xmlconfig` instead of `self.loadZCML`)? Is `self.loadZCML` maybe not available because `plone.testing` is too low level?

Reply:
For my taste I would put each argument on a line, but either is good.

Reply:
@mauritsvanrees You are right, using line breaks allows to omit ignoring `E501`. I added a comment why `D001` has to be ignored. Maybe this is a smell that `D001` is a bit too Plone specific. But there are other checks in `D001` which could even make sense here so I did not completely ignore it.